server/checks/PullRequestLabels.js

```diff
@@ -19,11 +19,31 @@ export function generateStatus(labels, checkConfig) {
   if (missingLabels.size > 0) {
     return createStatePayload(`PR misses required labels: ${[...missingLabels].join(', ')}.`, 'failure')
   }
+  checkAdditionalLabels(setDifference(labelSet, requiredSet), additional)
```

Comment:
You are missing the `return` here as otherwise the `status` is not returned.

Reply:
thx for spotting, added